recipes/recipes_emscripten/sparrow/recipe.yaml

```diff
@@ -0,0 +1,41 @@
+context:
+  version: 0.3.0
+
+package:
+  name: sparrow
+  version: ${{ version }}
+
+source:
+  url: https://github.com/man-group/sparrow/archive/${{ version }}.tar.gz
+  sha256: a520455e0def42c2d36f978a83d4157dc93c4f3737474f83c78e854664a1abb2
+
+build:
+  number: 0
+
+requirements:
+  build:
+  - ${{ compiler("cxx") }}
+  - cmake
+  - make    # [unix]
+  host:
+  - howardhinnant_date
+  run:
+  - howardhinnant_date
```

Comment:
Guessing we need the date library at runtime too !

Reply:
It is not required, the date library has [run_exports](https://github.com/conda-forge/howardhinnant_date-feedstock/blob/main/recipe/meta.yaml#L19-L20)

Reply:
Shall take care in the next update!
Thanks for the comment